kasa/discover.py

```diff
@@ -72,6 +72,7 @@ def datagram_received(self, data, addr) -> None:
 
         device_class = Discover._get_device_class(info)
         device = device_class(ip)
+        asyncio.ensure_future(device.update())
```

Comment:
This changes behavior, so that there is no need to separately call update on the discovered devices, which is reasonable, I think.